source/beamline/AMPVControl.cpp

```diff
@@ -459,8 +483,8 @@ void AMPVwStatusControl::onReadPVInitialized()
 }
 
 void AMPVwStatusControl::onWritePVInitialized() {
-	lowLimitValue_ = writePV_->lowerControlLimit();
-	highLimitValue_ = writePV_->upperControlLimit();
```

Comment:
Unless I'm missing something here, you call `setLowLimitValue()` here and inside AMPVwStatusAndUnitConversionControl those methods only emit a signal.  They overload the standard call and only emit the signals and not update the `lowLimitValue_` and `highLimitValue_` members.